client/src/App.css

```diff
@@ -47,27 +38,33 @@
   margin-right: 30px;
 }
 .canvasContainer {
+  border-radius: 25px;
+  border: 2px solid #21ad4b;
   position: relative;
-  width: 800px;
-  height: 533px;
-  padding-right: 20px;
-  padding-left: 20px;
-  padding-top: 20px;
-  padding-bottom: 20px;
+  width: 65vh;
+  height: 65vh;
```

Comment:
@mahadevans87 : I think it should be `vw` for width. Or has `vh` been used intentionally ?

Reply:
vh is intentional da. For portraits we define a square aspect ratio with vh. For landscapes (tablets, desktops), we define a square aspect ratio for vw